aten/src/ATen/Context.cpp

```diff
@@ -122,6 +122,14 @@ void Context::setAllowTF32CuDNN(bool b) {
   allow_tf32_cudnn = b;
 }
 
+bool Context::allowTF32Onednn() const {
+  return allow_tf32_onednn;
+}
+
+void Context::setAllowTF32Onednn(bool b){
```

Comment:
@ZhiweiYan-96 , Guanye's comment is marked as resolved. Do you mean it is not applicable?

Reply:
No, the comments here is same as the one above. I responded in the above comments. The naming here is aligned to @yanbing-j 's style.